content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -12,7 +12,7 @@ This section details some ways to simplify the management of dependency versions
 
 == Jenkins Core BOM
 
-Since version 2.195, Jenkins Core provides a  link:https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies[Maven Bill Of Materials (BOM)] that centrally defines versions of various libraries used by Jenkins.
+Since version 2.195 (and LTS version 2.164.x) Jenkins Core provides a  link:https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies[Maven Bill Of Materials (BOM)] that centrally defines versions of various libraries used by Jenkins.
```

Comment:
Maybe clarify: LTS versions preceding the 2.204.x timeline.

Reply:
do you have a phrasing suggestion?

Reply:
Best to ask @jtnord for details.

Note that https://github.com/jenkinsci/plugin-pom#introduction says 2.200+ whereas https://github.com/jenkinsci/plugin-pom#jenkins-core-bom says 2.195+. Not sure we really care about documenting weekly baselines prior to known LTS lines.

Reply:
I recall having to fix an issue in the weekly bit history shows nothing.

could just as easily say every LTS release between 2.164 and 2.204 and every release since.  but I doubt anyone cares that much